src/views/RecIM/views/Team/index.jsx

```diff
@@ -78,31 +83,53 @@ const Team = () => {
     setOpenTeamForm(false);
   };
 
+  const handleDelete = () => {
+    deleteTeam(teamID);
+    setOpenConfirmDelete(false);
+    setOpenSettings(false);
+    navigate(`/recim/activity/${team.Activity.ID}`);
+    // @TODO add snackbar
+  };
+
   if (!profile) {
     return loading ? <GordonLoader /> : <GordonUnauthorized feature={'the Rec-IM page'} />;
   } else {
     let headerContents = (
       <Grid container direction="row" alignItems="center" columnSpacing={4}>
-        <Grid item>
-          <img src={team?.Logo ?? defaultLogo} alt="Team Icon" width="85em"></img>
-        </Grid>
-        <Grid item xs={8} md={5}>
-          <Typography variant="h5" className={styles.title}>
-            {team?.Name ?? <GordonLoader size={15} inline />}
-            {hasPermissions && (
-              <IconButton
-                onClick={() => {
-                  setOpenTeamForm(true);
-                }}
-                className={styles.editIconButton}
-                sx={{ ml: 1 }}
-              >
-                <EditIcon />
-              </IconButton>
-            )}
-          </Typography>
-          {teamRecord()}
+        <Grid item container xs={9} columnSpacing={4} direction="row" alignItems="center">
+          <Grid item>
+            <img src={team?.Logo ?? defaultLogo} alt="Team Icon" width="85em"></img>
+          </Grid>
+          <Grid item>
+            <Typography variant="h5" className={styles.title}>
+              {team?.Name ?? <GordonLoader size={15} inline />}
+              {hasPermissions && (
+                <IconButton
+                  onClick={() => {
+                    setOpenTeamForm(true);
+                  }}
+                  className={styles.editIconButton}
+                  sx={{ ml: 1 }}
+                >
+                  <EditIcon />
+                </IconButton>
+              )}
+            </Typography>
+            {teamRecord()}
```

Comment:
```suggestion
            {teamRecord()}
```
This unfortunately doesn't work the way we want it to. I'll rework it in a separate PR unless you want to.

The issue lies in the fact that teamRecord pulls from the first available series' record. Which is not an accurate team record. There are 2 ways we could potentially handle this:

1. We can do a summation on ALL the series records and make a global Activity Record (which might make some teams appear worse because they made the playoffs but didn't do well) This is an easy fix, just do a foreach and add them together
2. **my preference** We implement a way to switch between matches of a series. (could be by tabs or some other way) Assume we use tabs, if I select `series#1` I'll get my `series#1` record and Schedule will show the matches of that series. This also means that we would need to have a `Match History` implementation on the Team Page that shows previous games which might also need to be sorted. This is relatively complex but I feel would be the coolest. (Possibly something to work on down the line)

Reply:
Regardless, non-blocking

Reply:
I just realized this wasn't actually a part of your PR rather just reformatting. NVM but I think the logic still stands and we probably need to make an issue for it.